library/src/trips/index.ts

```diff
@@ -64,8 +59,7 @@ const findCities = (
 
   const words = query.split(" ");
   const wordsCount = words.length;
-
-  if (wordsCount < 2) {
+  if (wordsCount < 2 || words[1]?.trim()?.length === 0) {
```

Comment:
```suggestion
  if (wordsCount < 2 || words[1].trim().length === 0) {
```

The `?.` confused me as if they were useful, this would return `null` and not 
`0`: I thing we shouldn't use `?` when not necessary.

And "because of TypeScript" isn't a good reason 😬 (another example of things I don't like in TS…)

Reply:
Yeah, indeed they are not useful here.